packages/shared/friends/sagas.ts

```diff
@@ -1169,13 +1172,27 @@ function* handleSendPrivateMessage(action: SendPrivateMessage) {
 }
 
 function* handleUpdateFriendship({ payload, meta }: UpdateFriendship) {
-  const { action, userId, future } = payload
+  const { action, userId, future, messageBody } = payload
 
   const client: SocialAPI | undefined = yield select(getSocialClient)
 
   // Get feature flag value
   const newFriendRequestFlow = isNewFriendRequestEnabled()
 
+  // Get renderer modules
+  const rendererModules: RendererModules | undefined = yield select(getRendererModules)
+  if (!rendererModules) {
+    yield call(future.resolve, { userId: userId, error: FriendshipErrorCode.FEC_UNKNOWN })
```

Comment:
Simplifying here and below.
```suggestion
    yield call(future.resolve, { userId, error: FriendshipErrorCode.FEC_UNKNOWN })
```